pkg/chains/storage/tekton/tekton.go

```diff
@@ -51,8 +52,8 @@ func NewStorageBackend(ps versioned.Interface, logger *zap.SugaredLogger) *Backe
 }
 
 // StorePayload implements the Payloader interface.
-func (b *Backend) StorePayload(ctx context.Context, tr *v1beta1.TaskRun, rawPayload []byte, signature string, opts config.StorageOpts) error {
-	b.logger.Infof("Storing payload on TaskRun %s/%s", tr.Namespace, tr.Name)
+func (b *Backend) StorePayload(ctx context.Context, clientSet versioned.Interface, obj objects.TektonObject, rawPayload []byte, signature string, opts config.StorageOpts) error {
```

Comment:
Instead of passing the client through as a param (which then needs to be passed to every other backend impl even though they don't use it) - why not use the existing `b.pipelineclientset`?

Reply:
Same for the other storage functions.